src/commands/dhcp_proxy.rs

```diff
@@ -55,6 +55,8 @@ struct NetavarkProxyService<W: Write + Clear> {
     cache: Arc<Mutex<LeaseCache<W>>>,
     // the timeout for the dora operation
     dora_timeout: u32,
+    // inactivity_timeout duration in seconds
+    inactivity_timeout: Duration,
```

Comment:
Would it be cleaner to wrap timeout_sender in an Option<> and then the option is set to None when the timeout is zero?
Looking at the code there should not be a reason to create the channel at all when the timeout is zero I think.